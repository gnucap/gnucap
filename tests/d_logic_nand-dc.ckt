* nor-test
V2 1 0 10V
V1 4 0 10V
U2 2 3 5 CMOS NOR
V3 5 0 10V
U1 3 1 2 CMOS NOR
.model CMOS LOGIC
.print dc V(V2) V(V3) V(U1) V(U2) logic(u*)
.dc V2 0 10 2
.status notime
.END
