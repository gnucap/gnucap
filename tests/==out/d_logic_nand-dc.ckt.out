* nor-test 
U2: can't find subckt: CMOSnor2, forcing digital
U1: can't find subckt: CMOSnor2, forcing digital
#           V(V2)      V(V3)      V(U1)      V(U2)      logic(U2)  logic(U1) 
 0.         0.         10.        5.         0.         0.         3.        
 2.         2.         10.        5.         0.         0.         3.        
 4.         4.         10.        5.         0.         0.         2.        
 6.         6.         10.        0.         0.         0.         0.        
 8.         8.         10.        0.         0.         0.         0.        
 10.        10.        10.        0.         0.         0.         0.        
Gnucap   System status
iterations: op=0, dc=20, tran=0, fourier=0, total=20
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=5, subckt=0, model=0, total=5
dctran density=20.0%, ac density=20.0%
